### PR TITLE
Introduce version manager for automatic version releases

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: kebechet/managers/version/version.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+_LOGGER = logging.getLogger(__name__)
+_VERSION_REQUEST_ISSUE = ' release'
+_VERSION_PULL_REQUEST_NAME = 'Release of version {}'
+
+
+class VersionManager(ManagerBase):
+    """Automatic version management for Python projects."""
+
+    def _adjust_version_in_sources(self, version_identifier: str):
```
Comment (goern) [inline kebechet/managers/version/version.py]: 👍

File: kebechet/source_management.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def open_merge_request(self, commit_msg: str, branch_name: str, body: str, label
         else:
             raise NotImplementedError
 
-        merge_request.labels = set(labels)
+        merge_request.labels = set(labels or [])
```
Comment (goern) [inline kebechet/source_management.py]: nice! this is short for `if labels == None then labels = []` ?

Comment (fridex) [reply kebechet/source_management.py]: yes :)

Comment (fridex) [reply kebechet/source_management.py]: It's a python idiom

**Pull request merged**